lib/mayaUsd/nodes/layerManager.cpp

```diff
@@ -343,10 +355,10 @@ void LayerDatabase::prepareForWriteCheck(bool* retCode, void*)
     }
 }
 
-bool LayerDatabase::getStagesToSave()
+bool LayerDatabase::getStagesToSave(bool isExport)
 {
     auto allStages = MayaUsd::ufe::getAllStages();
-    bool checkSelection = (MFileIO::kExportTypeSelected == MFileIO::exportType());
+    bool checkSelection = isExport && (MFileIO::kExportTypeSelected == MFileIO::exportType());
```

Comment:
We only want to check selection if we are in Export and the export mode is Export Selected.